frontend/model/contracts/chatroom.js

```diff
@@ -154,6 +100,7 @@ sbp('chelonia/defineContract', {
             maxDescriptionLength: CHATROOM_DESCRIPTION_LIMITS_IN_CHARS
           },
           attributes: {
+            adminIDs: [],
```

Comment:
If this PR requires recreating groups, then it is not necessary to do anything about this.

However, if the plan is to support existing groups, then this must have corresponding code in `postUpgradeVerification`

Reply:
Thanks for bringing this up. Although this PR in itself shouldn't require recreating groups, it'll probably be required at some point in the future, as references to global getters will fail. However, another option would be replacing the only accessor to `adminIDs` with an optional chain (`adminID?.includes`)

Reply:
Or actually, scractc that. No change is needed for the same intended effect (delete will fail with the wrong error message if a non-admin does it). I'd also avoid having to define an action that sets the admin, lest it be used by someone who isn't an admin, at least until there's a supported way of doing this. So, let's consider this a breaking change.

Reply:
Options then are:

1. Leave this as is. Admin using this action will get a generic JS error.
2. Use optional chain. Admin won't be able to use this action but will get proper error message.
3. Add update action (downside is that anyone can set themselves as admin). Maybe we can go for this, and then remove this action once groups are re-created.

I don't think this PR is breaking in itself (although that I haven't tried) other than for this `adminIDs` thing. However, it's not a forward-compatible change (meaning that a _future_ version of Chelonia won't support _current & past_ contracts, since at that point `sbp('state/vuex/getters)` will not be defined).

Reply:
OK, how about a write-once action that sets the chatroom admin if the admin isn't already defined.

This way, once the action is used, it cannot be used again to modify the admin.